R/FilteredDataset.R

```diff
@@ -108,15 +111,16 @@ FilteredDataset <- R6::R6Class( # nolint
     #' @description
     #' Gets a filter expression.
     #'
-    #' This functions returns filter calls equivalent to selected items
+    # TODO - explain this better
```

Comment:
> # TODO tags are addressed (or discarded)

(at least) one more

Reply:
I was hoping @gogonzo could pitch in on this one.